docs/_static/custom.css

```diff
@@ -0,0 +1,4 @@
+div#python2-eol {
```

Comment:
Looks like this is only used in the "admonition" below, maybe we can lose this file too.

Reply:
Done.